README.md

```diff
@@ -1,59 +1,111 @@
 # invenio-subjects-mesh
 
-MeSH subject terms for InvenioRDM
+*MeSH topical subject terms with qualifiers for InvenioRDM*
+
+Install this extension to get Medical Subject Headings topics with qualifiers into your instance.
+
+If you are looking for a smaller MeSH vocabulary without the qualifiers, use
+[invenio-subjects-mesh-lite](https://github.com/galterlibrary/invenio-subjects-mesh-lite) instead.
+
+> [!NOTE]
+> Both extensions map the `MeSH` subject id, so you can only install one of them. However, they
+> use the same pattern for subject ids, so you can switch from one to the other easily if your
+> needs change in the future.
 
-Install this extension to get Medical Subject Headings topics into your instance.
 
 ## Installation
 
 From your instance directory:
 
-    pipenv install invenio-subjects-mesh
+```bash
+pipenv install invenio-subjects-mesh
+```
 
 This will add it to your Pipfile.
 
-### Versions
+## Versions
 
-This repository follows [calendar versioning](https://calver.org/):
+This repository follows [calendar versioning](https://calver.org/) for year and month:
```

Comment:
Noticed this after reviewing another PR, but the version includes month? Should this be `2023.11.2` and not `2023.1.2`?

Reply:
The `2023.1.2` is actually correct - I haven't updated the vocabulary list with MeSH terms from this month. Those are still the terms from 2023-01 . The `2` is just for a patch. Thanks for the double-check though!